### PR TITLE
Final commit

### PR DESCRIPTION
**Pull request by arinzeatusiuba**
This is my task 6 for lessons 1 and 2, please review @ChidiPatrick @promise-J @OgJaykay
@Pieta9221 @MiniHack @Melumm
please kindly note that our instructor asked that I avoid the lint error check

### CODE REVIEW

**Review by Pieta9221 — COMMENTED** [review]
:+1: :+1: :+1:
Kindly fix lint error, give a little space between each input field, merge when you are done

File: html/index.html [hunk 1]
```diff
@@ -11,35 +11,35 @@
     <form method="post" action="#">
       <h2>Registration Form</h2>
       <label for="name">Name:</label>
-      <input type="text" id="name" name="name" required /><br />
+      <input type="text" id="name" name="name" required /><br /> <br>
```
Comment (Pieta9221) [inline html/index.html]: Great
Although, CSS margin property can also do the job and saves you time of writing multiple code

**Comment by Pieta9221** [comment]
Good job 
Go ahead and merge

File: html/index.html (hunk 1 above)
Comment (arinzeatusiuba) [reply html/index.html]: Oh true, Thank you so much @Pieta9221

**Pull request merged**